### PR TITLE
[PROCEDURES] Clarify how +1 votes on PRs can be expressed 

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Review by dannon — APPROVED** [review]

**Review by jmchilton — APPROVED** [review]

File: doc/source/project/organization.rst [hunk 1]
```diff
@@ -120,9 +120,18 @@ Everyone is encouraged to express opinions and issue non-binding votes on pull
...
+Votes on pull requests should take the form of +1, 0, -1, and fractions as
+outlined by the `Apache Software Foundation voting rules`_. The following are
+equivalent to a +1 vote:
+
+- a `thumbs up reaction <https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/>`__
+  on the pull request description;
+- approving the pull request when submitting a
+  `review <https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/>`__.
+
+Tha latter is the preferred method because it is integrated in GitHub, it allows
```
Comment (mvdbeek) [inline doc/source/project/organization.rst]: s/Tha/The

**Review by mvdbeek — APPROVED** [review]

File: doc/source/project/organization.rst (hunk 1 above)
Comment (nsoranzo) [reply doc/source/project/organization.rst]: Fixed, thanks!

**Comment by dannon** [comment]
This needs to be closed and re-opened as a fresh PR for actual voting, since it has been modified, for what it's worth.  (I think I tried to fix this bug/feature in our procedures a long while back, but failed)

**Comment by nsoranzo** [comment]
@dannon Sure, will do. But I'll add another commit to allow for spelling mistakes to be fixed without closing/reopening these PRs in the future.

**Comment by dannon** [comment]
@nsoranzo That sounds great to me, thanks!

**Pull request closed without merging**